src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs

```diff
@@ -152,20 +149,6 @@ public enum IsolatedStorageScope
         Roaming = 8,
         User = 1,
     }
-
-    public enum IsolatedStorageSecurityOptions
```

Comment:
Ok, I've removed them. I was originally going to leave the trivial implementations, but I don't have a great argument for keeping them.